storage/scanner.go

```diff
@@ -125,28 +125,35 @@ func (rs *rangeScanner) RemoveRange(rng *Range) {
 	rs.removed <- rng
 }
 
+// heartBeat returns a duration between iterations to allow us to pace
+// the scan.
+func (rs *rangeScanner) heartBeat(start time.Time) time.Duration {
```

Comment:
Actually I don't think "heartbeat" is the right word for this function. Maybe `paceInterval`?